editor/src/clj/editor/app_view.clj

```diff
@@ -1320,23 +1320,32 @@ If you do not specifically require different script states, consider changing th
     (console/pipe-log-stream-to-console! in)
     (PipedOutputStream. in)))
 
+(defn- build-html5! [project prefs web-server build-errors-view changes-view main-stage tool-tab-pane pre-bob-cmds]
+  (let [main-scene (.getScene ^Stage main-stage)
+        render-build-error! (make-render-build-error main-scene tool-tab-pane build-errors-view)
+        render-reload-progress! (make-render-task-progress :resource-sync)
+        render-save-progress! (make-render-task-progress :save-all)
+        render-build-progress! (make-render-task-progress :build)
+        task-cancelled? (make-task-cancelled-query :build)
+        bob-args (bob/build-html5-bob-options project prefs)
+        out (start-new-log-pipe!)]
+    (build-errors-view/clear-build-errors build-errors-view)
+    (disk/async-bob-build! render-reload-progress! render-save-progress! render-build-progress! out task-cancelled?
+                           render-build-error! (vec (concat pre-bob-cmds bob/build-html5-bob-commands)) bob-args project changes-view
+                           (fn [successful?]
+                             (when successful?
+                               (ui/open-url (format "http://localhost:%d%s/index.html" (http-server/port web-server) bob/html5-url-prefix)))
+                             (.close out)))))
+
+(handler/defhandler :rebuild-html5 :global
+  (enabled? [] (not (build-in-progress?)))
+  (run [project prefs web-server build-errors-view changes-view main-stage tool-tab-pane]
+       (build-html5! project prefs web-server build-errors-view changes-view main-stage tool-tab-pane
+                     ["clean" "clear" "resolve"])))
```

Comment:
This is the first time I see `clear` command...

Reply:
😳 how it's end up in my head and then here? O_o
I'll replace these two with "distclean"